image.c

```diff
@@ -138,7 +145,7 @@ bool img_load_gif(img_t *img, const fileinfo_t *file)
 	if (img->multi.cap == 0) {
 		img->multi.cap = 8;
 		img->multi.frames = (img_frame_t*)
-		                    emalloc(sizeof(img_frame_t) * img->multi.cap);
+							emalloc(sizeof(img_frame_t) * img->multi.cap);
```

Comment:
What's going on here? I don't see any reason to change indentation. There's a couple more places where indentation was changed.

I think we should keep indentation as is, if we are to change it, it should be changed throughout the codebase and should be its own commit/PR.

Reply:
that was done by ytrew, i opened this on a "as is" state to get feedback and correct these kind of things, will go through it later and make sure the indentation is consistent.